### PR TITLE
added pg:wait

### PR DESCRIPTION
**Pull request by jdx**

### CODE REVIEW

**Comment by jdx** [comment]
@uhoh-itsmaciek this one is ready for review

File: commands/wait.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+function * run (context, heroku) {
+  const fetcher = require('../lib/fetcher')
+  const host = require('../lib/host')
+  const app = context.app
+  const db = context.args.database
+
+  let waitFor = co.wrap(function * waitFor (db) {
+    const wait = require('co-wait')
+    let interval = parseInt(context.flags['wait-interval'])
+    if (!interval || interval < 0) interval = 1
```
Comment (msakrejda) [inline commands/wait.js]: Is this what we default to now? That seems rather low, considering that the average wait is probably a couple of minutes at least (for fresh databases) or up to hours (for forks and followers).

File: commands/wait.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+      cli.action.status(status.message)
+
+      yield wait(interval * 1000)
+    }
+  })
+
+  let dbs = []
+  if (db) dbs = yield [fetcher.addon(heroku, app, db)]
+  else dbs = yield fetcher.all(heroku, app)
```
Comment (msakrejda) [inline commands/wait.js]: Does the style guide allow rampant omission of braces like this? =D

**Comment by msakrejda** [comment]
A couple of comments, but +1. Sorry this fell of my radar.

File: commands/wait.js (hunk 2 above)
Comment (jdx) [inline commands/wait.js]: heh I could probably wrap these

File: commands/wait.js (hunk 1 above)
Comment (jdx) [inline commands/wait.js]: it is what the default is now, maybe bump it to 5-10 seconds?

Comment (msakrejda) [inline commands/wait.js]: Yeah, that (either of these, I guess 5 if you want an opinion but I'm just pulling that out of thin air) sounds more reasonable.

**Comment by jdx** [comment]
thanks!

**Comment by msakrejda** [comment]
Sweet, +1

**Pull request merged**